src/app/operations/operation-path.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, OnInit, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-operation-path',
+  template: `
+    <pre class="definition"><code><span class="http_method {{methodType}}">{{methodType}}</span><span class="path">{{path}}</span></code></pre>
+  `,
+  styles: []
+})
+export class OperationPathComponent implements OnInit {
```

Comment:
Is there a reason you're importing and implementing `OnInit` on every class?  Seems like something we'd want to avoid unless we explicitly need it since it's basically an extra event listener on everything that implements it.